sound/soc/sof/amd/acp-loader.c

```diff
@@ -144,9 +144,10 @@ static void configure_pte_for_fw_loading(int type, int num_pages, struct acp_dev
 int acp_dsp_pre_fw_run(struct snd_sof_dev *sdev)
 {
 	struct pci_dev *pci = to_pci_dev(sdev->dev);
+	struct snd_sof_pdata *plat_data = sdev->pdata;
```

Comment:
should this patch be a fixup if it is not upstream? if not, should there be a fixes tag?